python/python/glide/async_commands/server_modules/ft.py

```diff
@@ -54,3 +54,25 @@ async def create(
         for field in schema:
             args.extend(field.toArgs())
     return cast(TOK, await client.custom_command(args))
+
+
+async def dropindex(client: TGlideClient, indexName: TEncodable):
+    """
+    Drops an index. The index definition and associated content are deleted. Keys are unaffected.
+
+    Args:
+        client (TGlideClient): The client to execute the command.
+        indexName (TEncodable): The index name for the index to be dropped.
+
+    Returns:
+        If the index is successfully dropped, returns "OK" else throws an error
```

Comment:
This is the same as saying: 
If the command doesn't throw an error, returns "Ok".  

Do we need that if statement for every command?

Reply:
We do need to indicate what is the meaning of "OK". If we just say returns OK, there is no indication about what has been returned. OK that function ran of index dropped/created, etc.

Other than "OK" it is implied by default that errors will be thrown if something went wrong. That is the case for every command. This is too much detail for no reason.

Reply:
Got the statement to write from the documentation. https://docs.aws.amazon.com/memorydb/latest/devguide/vector-search-commands-ft.dropindex.html
Returns a simple string OK message or an error reply.